cmd/link.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/redhat-developer/odo/pkg/application"
+	"github.com/redhat-developer/odo/pkg/component"
+	"github.com/redhat-developer/odo/pkg/project"
+	"github.com/spf13/cobra"
+)
+
+var (
+	linkComponent string
+)
+
+var linkCmd = &cobra.Command{
+	Use:   "link <target component> --component [source component]",
+	Short: "Link target component to source component",
+	Long: `Link target component to source component
+If source component is not provided, the link is created to the current active
+component.
+In the linking process, the environment variables containing the connection
+information from target component are injected into the source component and
+printed to STDOUT.
+`,
+	Example: `
+	# Link current component to a component 'nodejs'
```

Comment:
There should be just two spaces at the beginning of each line, to make it properly align

Reply:
fixed